packages/template-retail-react-app/app/components/_app-config/index.jsx

```diff
@@ -85,9 +85,17 @@ AppConfig.restore = (locals = {}) => {
 
 AppConfig.freeze = () => undefined
 
+AppConfig.extraGetPropsArgs = (locals = {}) => {
+    return {
+        buildUrl: locals.buildUrl,
+        site: locals.site,
+        locale: locals.locale
+    }
+}
+
```

Comment:
Probably the result of a merge conflict. But this code shouldn't exist anymore in v3.

Reply:
Thanks! Had a difficult merge where I had to weave "left" and "right" in a bunch of spots